CHANGELOG.md

```diff
@@ -15,6 +15,99 @@ and includes an additional section for migration notes.
 
 
 ## [Unreleased]
+
+
+### Added
+- *ORCA-161* Added dead letter queue and cloudwatch alarm terraform code to recovery SQS queue.
+
+### Changed
+- *ORCA-161* Changed staged recovery SQS queue type from FIFO to standard queue.
+
+### Migration Notes
+
+- The user should update their `orca.tf`, `variables.tf` and `terraform.tfvars` files with new variable. The following required variable has been added:
+  - db_admin_username (defaults to "postgres")
+  
+- Add the following ORCA required variables definition to your `variables.tf` or `orca_variables.tf` file.
+
+```terraform
+variable "db_admin_password" {
+  description = "Password for RDS database administrator authentication"
+  type        = string
+}
+
+variable "db_user_password" {
+  description = "Password for RDS database user authentication"
+  type        = string
+}
+
+variable "db_host_endpoint" {
+  type        = string
+  description = "Database host endpoint to connect to."
+}
+
+variable "rds_security_group_id" {
+  type        = string
+  description = "Cumulus' RDS Security Group's ID."
+}
+
+variable "dead_letter_queue_topic_subscription_email" {
+  type        = string
+  description = "The email to notify users when messages are received in dead letter SQS queue"
```

Comment:
Should just need this part. Might want to emphasize that the queue is filled by failed restore requests, and that emails will only happen once until the dead letter queue is emptied. Might be good to note that in general docs as well.

Reply:
ok I thought we needed to add all the required variables here. 
removed the other variables.

Added more info to the variable description like you suggested